debian/debian_test.go

```diff
@@ -95,11 +95,11 @@ func TestClient_Update(t *testing.T) {
 				Retry:       0,
 			}
 			err = client.Update()
-			if tc.expectedError == "" {
-				assert.Nil(t, err)
-			} else {
-				assert.Contains(t, err.Error(), tc.expectedError)
-				return
+			switch {
+			case tc.expectedError != "":
+				assert.Contains(t, err.Error(), tc.expectedError, tc.name)
+			default:
+				assert.NoError(t, err, tc.name)
```

Comment:
Why did you replace `if` with `switch`?

Reply:
Just personal preference. Looks a bit cleaner IMO.

Reply:
I see.